ITEST/itest.webstudio/test-resources/admin/tag-config/create-tag-1.post.req

```diff
@@ -0,0 +1,4 @@
+POST /web/admin/tag-config/types/1/tags HTTP/1.1
+Content-Type: application/json; charset=UTF-8

```

Comment:
Why content type is json, if you are passing a regular text?

Reply:
Fixed to use text/plain